src/theme/theme.ts

```diff
@@ -50,6 +51,8 @@ const theme: ITheme = {
 
     black: '#272727',
     white: '#FFFFFF',
+    error: '#F8DEDE',
+    warning: '#FDF0E4',
 
```

Comment:
name them  `redShade3` and `orangeShade3`
and include them the label part below